### PR TITLE
Fixes #23607 - Add the UI/UX team as a code-owner

### PR DESCRIPTION
**Pull request by sharvit**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,7 @@
 Gemfile         @theforeman/packaging
-Rakefile.dist	@theforeman/packaging
+Rakefile.dist	  @theforeman/packaging
```
Comment (amirfefer) [inline .github/CODEOWNERS]: could you remove the redundant space here?

**Review by amirfefer — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,4 +1,7 @@
 Gemfile         @theforeman/packaging
-Rakefile.dist	@theforeman/packaging
+Rakefile.dist   @theforeman/packaging
 bundler.d/*.rb  @theforeman/packaging
 package.json    @theforeman/packaging
+package.json    @theforeman/ui-ux
```
Comment (ekohl) [inline .github/CODEOWNERS:5]: I think this doesn't work. https://help.github.com/articles/about-codeowners/ says it should be `package.json @theforeman/packaging @theforeman/ui-ux`

**Review by ekohl — CHANGES_REQUESTED** [review]
Remember this only works if the ui-ux team has write permissions on the repository. I don't know if all members already have write access but it's something to be aware of. Usually we go through the nomination process if there are new members.

**Comment by sharvit** [comment]
OK, we just decided to give up :disappointed: 

Anyway, there is a @theforeman/ui-ux team you can mention from now on.

**Comment by ekohl** [comment]
> OK, we just decided to give up disappointed

I'm interested to know the reasons behind giving up.

**Comment by sharvit** [comment]
@tstrachota is the only one having write-access

We mostly wanted it so we can track UI/UX related PRs automatically.
We are going to achieve that by updating the mention-bot

**Comment by ekohl** [comment]
@sharvit I don't mind getting more people write-access (we did it for foreman-packaging too for similar reasons).

From experience I can also tell you that GH doesn't notify you about PRs that require your team - you can either watch the whole repository or nothing AFAIK. It's also something I struggled with and haven't found a good solution. Perhaps we should also update mention bot on the packaging side. Let me know what you come up with because we have the same issue we're trying to solve :)

**Comment by ohadlevy** [comment]
@sharvit I'm actually in favor of this, @tstrachota has access and so do I (which i still kind of consider related to this change). other maintainers may step up in the (near) future as well, so +1 for this.

File: .github/CODEOWNERS (hunk 2 above)
Comment (ohadlevy) [reply .github/CODEOWNERS:5]: I would also add webpack configuration files and the code under /scripts/ that handle the packaging / webpack integration.

**Comment by ekohl** [comment]
@ohadlevy if the team doesn't have write access Github ignores the team in CODEOWNERS. It's why we gave the packaging team write access on core. I think this might be a good thing to implement, but if we do implement it, let's make sure it actually works as intended :)

**Comment by Rohoover** [comment]
@sharvit 
Can we add Tereza Novotna to that UI/UX mention as well. She is in UXD and is now helping me out. Getting her involved in the upstream would be very beneficial in helping herself familiarize herself with the product.  @terezanovotna

**Pull request closed without merging**